github/ci/prow/files/jobs/kubevirt/project-infra/project-infra-periodics.yaml

```diff
@@ -200,6 +200,7 @@ periodics:
       - "autoowners"
       args:
       - --dry-run=false
+      - --assign=prow-job-taskforce
```

Comment:
Is this a real github-assign? Is prow-job-taskforce a real github group?

Reply:
https://github.com/orgs/kubevirt/teams/prow-job-taskforce

Reply:
Wouldn't this work with teams also? Do I need to manually adjust the commit comment then? Then I'd rather keep it like it is

Reply:
I can't assign it from the UI.

Reply:
No, you can't because we need to use this notation: @kubevirt/prow-job-taskforce